dcm4che-tool/dcm4che-tool-stowrs/src/main/java/org/dcm4che3/tool/stowrs/StowRS.java

```diff
@@ -200,7 +200,7 @@ private void stowMetaDataAndBulkData(File file) {
             try {
                 metadata = parseJSON(file.getPath());
             } catch (Exception e) {
-                LOG.error("error parsing metadata JSON file {}", file);
+                LOG.error("error parsing metadata JSON file {}: {}", file, e);
```

Comment:
For slf4j loggers will display the full exception, including stacktrace, even if you do not add the placeholder to the message itself:
`LOG.error("error parsing metadata JSON file {}", file, e);`
Please give that a try.